src/components/Link.js

```diff
@@ -9,11 +9,11 @@ import { Icon } from './Icon';
 import { RegularText } from './Text';
 import { WrapperRow } from './Wrapper';
 
-export const Link = ({ url, title }) => (
+export const Link = ({ url, descrption }) => (
```

Comment:
Have you checked, where we use the Link component, if it is possible to change the key here from `title` to `description`? This means, the props given needs to have a `description` key. If there were data with `title` before, this would not work anymore with the changed name. Be careful!
I have checked the code and we do not use this Link component currently.
So I will only fix the typo.